code/src/GridAPI/Rows.ts

```diff
@@ -15,20 +15,44 @@ namespace GridAPI.Rows {
         gridID: string,
         rowNumber: number,
         className: string
-    ): void {
+    ): string {
         PerformanceAPI.SetMark('Rows.AddClass');
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
```

Comment:
Please have an action, to be used cross API to do the set of this default object. Repeated with https://github.com/OutSystems/outsystems-datagrid-reactive/pull/188

Reply:
done